dstrunk/agnostic-test
# Add Vitest support, add tests

- Add Vitest support
- Add Jest tests for the test runner
- Add GitHub actions workflow for PRs to main to ensure tests are passing before merging
- Update CHANGELOG to reflect all changes made over the last couple of releases
